<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientsReception</class>
 <widget class="QWidget" name="PatientsReception">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fnameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="snameLabel">
     <property name="text">
      <string>Second name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="snameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ssnLabel">
     <property name="text">
      <string>SSN</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ssnEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dateLabel">
     <property name="text">
      <string>Date: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dateEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTableView" name="patientView"/>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="full_nameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
